tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -39,6 +39,26 @@ internal static class Debugger
             /// <seealso cref="DebuggerSettings.UploadBatchSize"/>
             public const string UploadBatchSize = "DD_DEBUGGER_UPLOAD_BATCH_SIZE";
 
+            /// <summary>
+            /// Configuration key for the maximum symbol size to upload (in bytes).
+            /// Default value is 1 mb.
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.SymbolBatchSizeInBytes"/>
+            public const string SymbolBatchSizeInBytes = "DD_DEBUGGER_SYMBOL_BATCH_SIZE_BYTES";
+
+            /// <summary>
+            /// Configuration key for allowing uploading symbol database.
```

Comment:
```suggestion
            /// Configuration key for allowing upload of symbol data (such as method names, parameter names, etc) to Datadog.
```

Reply:
Fixed in db6086b623f7ad8ae43181865be84140bce1ef3c